components/02-elements/search/search.hbs

```diff
@@ -0,0 +1,42 @@
+<div class="search-block">
+    <div class="search-block__toggle-btn">
+        <svg class="search__icon ">
+            <use xlink:href="/images/icons-sprite.svg#search"></use>
+        </svg>
+    </div>
```

Comment:
This button shouldn't be a part of `search` component. It should work same way as `side-menu` and `side-menu-trigger`